### ISSUE TITLE
not working after today's update

### ISSUE THREAD

**Issue by Sewer2K** [AmrHuss/MarvelRivals-cheat#16]
i added team check for aimbot and esp
but not working today after update, can you provide updated offsets / pointers for injection

**Comment by nobirdinflight** [comment]
not working too.

**Comment by Sewer2K** [comment]
yeah not sure what offsets or pointers to update offsets i can find from a sdk dump on unknown earlier today but the pointers im not sure 
i will respond here if i have any luck if you find something please lemme know too

**Comment by FlatManDud** [comment]
dump sdk yourself
https://github.com/Encryqed/Dumper-7

**Comment by Criskept1** [comment]
> danke will have a look in the mornning

Which injector do you use? When I inject mine, the hook success message appears but nothing happens when I press insert inside the game.

**Comment by Criskept1** [comment]
Does the menu appear for you when you press insert even though it is out of date today?
